xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -496,6 +496,21 @@ benchmarks! {
 		assert_eq!(executor.origin(), &Some(X1(alias).relative_to(&universal_location)));
 	}
 
+	export_message {
+		let (origin, network, destination) = T::export_message_origin_and_destination()?;
+		let mut executor = new_executor::<T>(origin);
+		// Actual exported message has no bearing on this instruction's weight, use empty message.
+		let inner_xcm = Xcm(vec![]);
```

Comment:
Looking at the code; this gets passed down to `validate_export` and in turn to `XcmSend::validate` which can have a weight dependency on the length of the message for example [here](https://github.com/paritytech/polkadot/blob/5a43bc733024b6f2d4f1164db8991a393c8e14cb/xcm/xcm-builder/src/universal_exports.rs#L345).

Could you maybe try to use a large `inner_xcm` here to ensure that we dont underestimate for large XCMs?